server/Cargo.toml

```diff
@@ -9,13 +9,28 @@ rust-version.workspace = true
 version.workspace = true
 
 [dependencies]
+base64 = "0.22"
 clap.workspace = true
 enumflags2 = "0.7"
+hkdf = { version = "0.12", optional = true }
+num = "0.4.0"
+num-bigint-dig = { version = "0.8", features = ["zeroize"] }
+openssl = { version = "0.10", optional = true }
 oo7 = { workspace = true, features = ["unstable"] }
 rpassword = "7.3"
 serde.workspace = true
+sha2 = { version = "0.10", optional = true }
 tokio = { workspace = true, features = ["full"] }
 tracing = "0.1"
 tracing-subscriber.workspace = true
 zbus.workspace = true
 zeroize.workspace = true
+
+[features]
+default = ["native_crypto"]
+native_crypto = [
+    "dep:hkdf",
+    "dep:sha2",
+]
+
+openssl_crypto = ["dep:openssl"]
```

Comment:
```suggestion
default = ["native_crypto"]
native_crypto = [
    "dep:hkdf",
    "dep:sha2",
    "oo7/native_crypto",
]

openssl_crypto = ["dep:openssl", "oo7/openssl_crypto"]
```

Reply:
Thanks! I tried this, but I'm getting `error: You can't enable both openssl_crypto & native_crypto features at once`. Also, tried disabling oo7 default features. It didn't work either.

Reply:
How are you building it for it to fail this way?

Reply:
`cargo build --no-default-features --features openssl_crypto`

Reply:
Fixed this with https://github.com/bilelmoussaoui/oo7/pull/206